pkg/k8s/api_test.go

```diff
@@ -4,26 +4,86 @@ import (
 	"testing"
 
 	"github.com/runconduit/conduit/pkg/shell"
+	"fmt"
+	"net/url"
 )
 
+type TestFixture struct{
```

Comment:
Could this be a `map[string]string`? Usually, by convention, the key is input and the value expected output.